docs-src/concepts/what-is-remote-data-encoding.md

```diff
@@ -10,28 +10,27 @@ tags:
 
 Remote data encoding is exposing your Payload Codec via HTTP endpoints to support remote encoding and decoding.
 
-Running your encoding remotely allows you to use it with `tctl` to encode/decode data for several commands including `tctl workflow start` and with Temporal Web UI to encode and decode data in your Workflow Execution details view.
+Running your encoding remotely allows you to use it with the [`temporal` CLI](/concepts/what-is-the-temporal-cli) to encode/decode data for several commands including `temproal workflow show` and with Temporal Web UI to decode data in your Workflow Execution details view.
```

Comment:
Typo: `temproal` == `temporal` (Is this something I could have fixed for you?)

Reply:
Yes, but good catch anyway :)